feature-gate/README.md

```diff
@@ -2,7 +2,8 @@
 
 This program serves to manage new features on Solana.
 
-It serves one purpose: revoking features that are pending activation.
+It serves two main purposes: activating new features and revoking features that
+are pending activation.
```

Comment:
```suggestion
It serves two main purposes: queuing new features for activation and revoking features that
are already queued and pending activation.
```
I'm not totally happy with my suggestion here (and I don't love adding another new term: "queued"), but I'm trying to express that the new instruction sets a feature to "pending". Thoughts?

Reply:
Yeah despite the fact it's still doing the same thing until SIMD 0066, I agree we should take the opportunity to describe it more clearly.

I think "queue" is the best I can think of in this context as well, so I'm good with it!